operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -808,11 +808,23 @@ message TelemetryV2PrometheusConfig {
   ConfigOverride config_override = 3;
 }
 
+// Types of Access logs to export.
+  enum AccessLogging {
```

Comment:
Should this enum be put into TelemetryV2StackDriverConfig? Seems to be sd only enum.

Reply:
done